frontend/screens/Homepage.js

```diff
@@ -218,6 +222,16 @@ const Homepage = ({ route, navigation }) => {
               }
             </ScrollView>
           </View>
+          <View style={styles.container}>
+      <View style={styles.chatContainer}>
+        <Text style={styles.subtitle}>
+          Community Chat
```

Comment:
this feature should only be accessible to logged in user right? otherwise it will led to spam

Reply:
yes sure ....
<img width="1434" alt="Screenshot 2024-07-18 at 1 53 08 PM" src="https://github.com/user-attachments/assets/02b4c101-ff40-4e59-bf6b-e215816d6444">